noxfile.py

```diff
@@ -61,7 +61,10 @@ def run_coverage(session):
     set_environment_variables(PYBAMM_ENV, session=session)
     session.install("coverage", silent=False)
     if sys.platform != "win32":
-        session.install("-e", ".[all,jax,odes]", silent=False)
+        if sys.version_info > (3, 12):
```

Comment:
Should this be `>=` or will `>` capture `3.12.0`?

Reply:
`>` is fine here – I had tested this locally on Python 3.12.0, we should be safe